lighthouse-core/test/gather/fetcher-test.js

```diff
@@ -7,20 +7,35 @@
 
 /* eslint-env jest */
 
+import {jest} from '@jest/globals';
+import Connection from '../../gather/connections/connection.js';
+import {fnAny, mockCommands} from '../test-utils.js';
+
+const {createMockSendCommandFn} = mockCommands;
+
+// Some imports needs to be done dynamically, so that their dependencies will be mocked.
+// See: https://jestjs.io/docs/ecmascript-modules#differences-between-esm-and-commonjs
+//      https://github.com/facebook/jest/issues/10025
+/** @typedef {import('../../gather/driver.js')} Driver */
+/** @type {typeof import('../../gather/driver.js')} */
+let Driver;
+/** @typedef {import('../../gather/fetcher.js')} Fetcher */
+/** @type {typeof import('../../gather/fetcher.js')} */
```

Comment:
`typeof Fetcher` doesn't work?

Reply:
It is circular, as Fetcher would refer to the variable and not the type/class in the line above.